osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -87,16 +87,15 @@ private void load(OsuColour colours, GameplayClock clock)
             };
         }
 
-        private const double skip_required_cutoff = 3000;
         private const double fade_time = 300;
+        private const int skip_required_time = -1000;
```

Comment:
Can you explain in xmldoc where this value comes from, and why it's negative?

Reply:
All this comment does is makes it sound like there should be a 2000 in a constant somewhere..?

Reply:
It does, but in mentioned file is just `- 2000` at var declaration. Shall I modify even there?

Reply:
It looked suspect for me, but I thought there must be a reason for that and I should not make changes over files that are not implied on the file that makes the bug / problem, also it was supposed to be already verified.

Reply:
Also this number CANNOT be below zero. That screams all kinds of wrong. You're implying users should be able to skip even when gameplay starts before the audio starts. The point of skip is to skip audio.